app/assemblers/assemble_exercise_representations_admin.rb

```diff
@@ -0,0 +1,37 @@
+class AssembleExerciseRepresentationsAdmin
+  include Mandate
+
+  def self.keys = %i[page order criteria track_slug]
+
+  initialize_with :params
+
+  def call
+    SerializePaginatedCollection.(
+      representations,
+      serializer: SerializeExerciseRepresentations,
+      serializer_kwargs: { params: },
+      meta: {
+        unscoped_total: Exercise::Representation.with_feedback.count
+      }
+    )
+  end
+
+  private
+  memoize
+  def representations
+    Exercise::Representation::Search.(
+      mentor: nil,
+      track:,
+      with_feedback: true,
+      criteria: params[:criteria],
+      page: params.fetch(:page, 1),
+      order: params.fetch(:order, :most_submissions)
+    )
+  end
+
+  def track
+    return if params[:track_slug].blank?
+
+    Track.find(params[:track_slug])
+  end
```

Comment:
I think this is wrong. I think it should require a track and blow up if one is missing.

Reply:
> The initial state is to show everything the person mentors unfiltered.

I don't think it is actually. The params passed in are: ```track_slug: params.fetch(:track_slug, track_slugs.first)``` (e.g. in `app/helpers/react_components/mentoring/representations/with_feedback.rb`). I might be wrong, but I don't think there's a scenario where we show everything anymore.